api/docker-compose.yaml

```diff
@@ -3,8 +3,8 @@ version: "3.4"
 services:
   db:
     image: mariadb:10.2
-    env_file:
-      - db/local-vars.env
+    env:
```

Comment:
this needs to be `environment`.

Fixing that only brings me back to my recent problems however:
```
Connection could not be created to jdbc:mysql://db:3306/cdr with driver com.mysql.jdbc.Driver
```